domains/risuu.json

```
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "wrisuu",
+        "discord_username": "wrisuu",
```

```suggestion
        "discord": "730160704409305088",
```
its better to use the Discord ID instead of username cause username can be change and we use this info to contact you.